src/main/java/com/example/controllers/TransferController.java

```diff
@@ -0,0 +1,145 @@
+package com.example.controllers;
+
+import java.security.Principal;
+import java.util.Date;
+import java.util.List;
+
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import javax.validation.Valid;
+
+import com.example.bean.Account;
+import com.example.bean.CashAccount;
+import com.example.bean.OperationConfirm;
+import com.example.bean.Transfer;
+import com.example.dao.AccountDao;
+import com.example.dao.CashAccountDao;
+import com.example.facade.TransfersFacade;
+import com.example.util.InsecureBankUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.CookieValue;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+@RequestMapping(value = "/transfer")
+public class TransferController {
+
+	private static final String PENDING_TRANSFER = "PENDING_TRANSFER";
+
+	@Autowired
+	CashAccountDao cashaccountDao;
+
+	@Autowired
+	AccountDao accountDao;
+
+	@Autowired
+	TransfersFacade transfersFacade;
+
+	@RequestMapping
+	public String newTransferForm(final Model model, final Principal principal, final HttpServletResponse response) {
+		List<CashAccount> cashAccounts = cashaccountDao.findCashAccountsByUsername(principal.getName());
+
+		Account account = accountDao.findUsersByUsername(principal.getName()).get(0);
```

Comment:
#### Automated PR Comment From Polaris SAST
<!-- 843e9d1a1873595cf236bc37765f533b -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Polaris SAST Issue - SQL Injection
_High CWE-89_
Untrusted user-supplied data is inserted into a SQL statement without adequate validation, escaping, or filtering.

A user can change the intent of the SQL query, which may inappropriately disclose or corrupt data within the database.

### How to fix
Rewrite all SQL queries constructed through dynamic concatenation to use an injection-safe query mechanism such as prepared statements with parameterized queries.

Most modern programming languages provide a feature called "parameterized queries" that allow user-supplied data to be inserted safely as values in dynamic SQL queries. Rather than construct the dynamic SQL query by concatenating user-supplied data to static SQL query string fragments, data values are identified in the query by parameter markers or variables. Dynamic data is then passed through a mechanism provided by SQL that prevents the supplied data from changing the meaning of the query.

Note: the exact syntax and use of prepared statements with parameterized queries vary from language to language.